wemake_python_styleguide/violations/best_practices.py

```diff
@@ -127,6 +129,8 @@
 .. autoclass:: FloatKeyViolation
 .. autoclass:: ProtectedModuleMemberViolation
 .. autoclass:: PositionalOnlyArgumentsViolation
+.. autoclass:: ContinueInFinallyBlockViolation
```

Comment:
`LoopControlFinallyViolation`?

Reply:
I have tried to name it `LoopControlInFinallyViolation` and created https://github.com/wemake-services/wemake-python-styleguide/issues/1191